egs/babel_multilang/s5/local/nnet3/run_shared_ivector_extractor.sh

```diff
@@ -67,7 +67,7 @@ fi
 
 if [ $stage -le 5 ]; then
   # To train a diagonal UBM we don't need very much data, so use the smallest subset.
-  steps/online/nnet2/train_diag_ubm.sh --cmd "$train_cmd" --nj 200 --num-frames 500000 \
+  steps/online/nnet2/train_diag_ubm.sh --cmd "$train_cmd" --nj 100 --num-frames 200000 \
```

Comment:
I didn't test this, but 500000 frames is very large and it takes long time to train ubm model!(200000 is the number of frames we use in fisher_swbd setup)
I will test it in small setup to see if it makes any difference.